app/src/main/java/com/fastaccess/provider/timeline/handler/QuoteHandler.java

```diff
@@ -19,9 +19,8 @@
 
     @ColorInt private int color;
 
+    @Override
     public void handleTagNode(TagNode node, SpannableStringBuilder builder, int start, int end) {
-        builder.append("\n");
```

Comment:
Look like we need this..

Reply:
the line starts too early now

Reply:
@yakov116 what do you mean? In current _development_ branch the test quotes are displayed as they should:

![screenshot_20180707-125735_fasthub debug](https://user-images.githubusercontent.com/18537755/42411403-bd8899f4-81fb-11e8-8b78-ee98149bcf1b.jpg)